index.js

```diff
@@ -113,4 +121,21 @@ io.on('connection', function (socket) {
             }
         }
     });
+    
+    // socket.on('disconnect', function () {
+    //     console.log("DISCONNECT FROM " + socket.id)
+       
+    //     for(var roomId in socket.rooms) {
```

Comment:
this is never getting entered, but if we end up using cookies or query strings then we probably wont need this anyway